spec/oktakit_spec.rb

```diff
@@ -46,13 +46,13 @@
     504 => Oktakit::ServerError,
   }
 
-  describe 'errors' do
-    ERRORS.each do |code, error|
-      it "raises a #{error} on #{code} responses" do
-        VCR.use_cassette(code) do
-          expect { client.get('/users/-1') }.to(raise_error(error))
-        end
-      end
-    end
-  end
+  # describe 'errors' do
```

Comment:
Let's change the tests to describe the actual behaviour, instead of commenting them out. Since this behaviour is already published and used, we can't safely change it now, so I'd say the tests should describe the actual instead of desired behaviour. 

I'd probably treat removing RaiseError as a breaking change, so this would go in oktakit 1.0 or something.